### PR TITLE
Xcode build version 0.0.2

### PR DESCRIPTION
**Pull request by TiagoBrasN**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

The test of this Step fails when it cannot find Info.plist and xcodeproj files.

Thanks for the changes suggested in the previous [PR](https://github.com/bitrise-io/bitrise-steplib/pull/2376#event-3078660099).

### CODE REVIEW

File: steps/xcode-build-version/assets/icon.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
...IECBAgAABAgQIzEHgtpnz2wlNkNkYvHEOa2pKAgQIECBAgAABAgQIECBAYBGBzbPvMwlNkGYNXhvTmyzibRcBAgQIECBAgAABAgQIECAwJ4HycZhPJzRBmjF4ZyzXTdgIECBAgAABAgQIECBAgACBjglsmHzeldAEmc7g/8bQOz+CYCNAgAABAgQIECBAgAABAl0VKH9x/38JTZDxDcrdXnzhaVdf2fIiQIAAAQIECBAgQIAAAQKLCOyRfecmNEJGMzg/Vrst4mgXAQIECBAgQIAAAQIECBAg0HGBeyS/LyU0QZY3+G6M7tPxtZQeAQIECBAgQIAAAQIECBAgsIzAejl2eOKKhEbIjQ3+Iy4bJ2wECBAgQIAAAQIECBAgQIDAAAS2SQ1nJDRB/mBQPvLy5AGsqxIIECBAgAABAgQIECBAgACBtQTWz/O/S1yUqLURUr7o9E2JWyZsBAgQIECAAAECBAgQIECAwIAF7pbajkjU1gT5XmreJWEjQIAAAQIECBAgQIAAAQIEKhLYNrV+PjH0RsgFqfF5iQ0TNgIECBAgQIAAAQIECBAgQKBSgVWp+6hE+XjIkJohZ6ee8pGfmyVsBAgQIECAAAECBAgQIECAAIHfCzw0//ueRN/vGPOj1PDcxGYJGwECBAgQIECAAAECBAgQIEBgUYHbZe/fJ76U6Ms7Qq5Krh9P7JUot/61ESBAgAABAgQIECBAgAABAgRGFrhnznxh4juJrjVDrklOn048J3H7hI0AAQIECBAgQIAAAQIECHRS4CadzEpSSwn8eQ48MvGIRPnekNsm2t5+mAk/dV2cmJ+/bDsB8xEgQIAAAQIECBAgQIAAgXEFNEDGFevW+X+WdLZJbJ24T6K8W+TWiaa28iWm30/8T6K80+OTibMSNgIECBAgQIAAAQIECBAg0CsBDZBeLddIyd4qZ22ZKM2QEjdNbH7dzzWP8/T3H6e5MD8vTVyeKI8vSfzguiiNj/LcRoAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIBA1QL/P07wgbQJyBE+AAAAAElFTkSuQmCC"/>
```
Comment (Szabadember) [inline steps/xcode-build-version/assets/icon.svg:14]: Please check the contribution guidelines, inline PNG images are not allowed.

**Comment by TiagoBrasN** [comment]
Thanks everyone. Deleted image from icon.svg.

**Comment by trapacska** [comment]
@TiagoBrasN could you please enable **Allow edits by maintainers**? You can read the guide to do that: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

We need to add a specific maintainer field to the repo that only our team members are eligible to add.

**Comment by trapacska** [comment]
@TiagoBrasN did you managed to turn it on?

**Comment by TiagoBrasN** [comment]
I can't turn it on on this account, I created a fork from my personal account. I'm going submit another a new PR with that option turned on. Thanks.

**Pull request closed without merging**